covid/extract_config/analysis_usgovcloudapi.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is a nitpick, but this file name is a little obscure to me. Maybe rename as something like `extract_config/constants.py`?  It doesn't matter too much though!

Reply:
Yeah, lack of creativity here. The contents are related to accessing `.analysis.usgovcloudapi.net` and that ended up being the name. Going to drop `analysis`, `us`, and `api` since those are inherent to the project